### PR TITLE
RS/YJ/get baseline system types add more systems

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR addresses adding more system types in the ```get_baseline_system_types``` function.

**Note)**
1. I'll add ```is_baseline_system_9``` once PR #891 is merged.
2. There are many section 22 RMD updates in this PR. The reason is rule 22-40 failed and I had to update its RMD. And I realized that the ```schema version``` keys wasn't updated in the past. So, I updated the keys.

### CODE REVIEW

**Comment by yunjoonjung-PNNL** [comment]
@charliepnnl FYI, this PR is ready to be reviewed. Thanks!

File: rct229/ruleset_functions/baseline_systems/is_baseline_system_2.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def is_baseline_system_2(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
         and are_all_terminal_heat_sources_none_or_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_cool_sources_none_or_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_fans_null(rmi_b, terminal_unit_id_list)
-        and are_all_terminal_supplies_ducted(rmi_b, terminal_unit_id_list)
+        and not are_all_terminal_supplies_ducted(rmi_b, terminal_unit_id_list)
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/is_baseline_system_2.py:79]: Oh wow.  I don't think I've been careful enough in my reviews to notice any of these functions calls needing to be False.  Have you  rechecked (or could you recheck) the other baseline_systems functions for this possibility?

File: rct229/ruleset_functions/get_baseline_system_types.py [hunk 2]
```diff
@@ -64,14 +103,87 @@ def get_baseline_system_types(rmi_b):
             len(dict_with_terminal_units_and_zones[terminal_id]) == 1
             for terminal_id in terminal_unit_id_list
         ):
-            hvac_sys = is_baseline_system_7(
```
Comment (charliepnnl) [inline rct229/ruleset_functions/get_baseline_system_types.py:67]: Just a thought:

We could remove some of the repetitiveness here with something like:
```
# Put this up high in the file
baseline_system_type_checks = [is_baseline_system_1, is_baseline_system_2, ...]
```
Then here, we could use
```
sys_found = False
for sys_check in baseline_system_type_checks:
    hvac_sys = sys_check(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list)
    if hvac_sys != HVAC_SYS.UNMATCHED:
        baseline_hvac_system_dict[hvac_sys].append(hvac_b_id)
        sys_found = True
        break
if not sys_found:
        # The error handling code goes here
```

**Review by charliepnnl — APPROVED** [review]
Just two comments. You may ignore them both if you choose.

File: rct229/ruleset_functions/baseline_systems/is_baseline_system_2.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/baseline_systems/is_baseline_system_2.py:79]: Of course, Charlie. I examined all other baseline_systems functions carefully, and I didn't find any function that we flipped the True/False.

File: rct229/ruleset_functions/get_baseline_system_types.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/get_baseline_system_types.py:67]: This is brilliant Charlie. I appreciate for letting me know about this.

This comment was addressed.

**Comment by yunjoonjung-PNNL** [comment]
@weilixu 

The ```fail``` cases were added in the unit test. As we discussed with each other, I'll make another update for system 3B after the difference between system 1B and 3B are identified.

**Comment by weilixu** [comment]
> @weilixu
> 
> The `fail` cases were added in the unit test. As we discussed with each other, I'll make another update for the system 3B unit test after the difference between system 1B and 3B are identified.

We will need to add `are_all_terminal_supplies_ducted` in system type 3.
I will modify the RDS in this file, once I am done, you can follow the modified RDS to update the system type 3.

**Review by weilixu — APPROVED** [review]

File: rct229/ruleset_functions/get_baseline_system_types.py [hunk 3]
```diff
@@ -64,45 +115,33 @@ def get_baseline_system_types(rmi_b):
...
-            hvac_sys = is_baseline_system_11_2(
-                rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list
-            )
-            if hvac_sys != HVAC_SYS.UNMATCHED:
-                baseline_hvac_system_dict[hvac_sys].append(hvac_b_id)
-                continue
+                if hvac_sys != HVAC_SYS.UNMATCHED:
+                    baseline_hvac_system_dict[hvac_sys].append(hvac_b_id)
+                    sys_found = True
+                    break
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/get_baseline_system_types.py:122]: This comment was addressed in PR #925.

File: rct229/ruleset_functions/get_baseline_system_types.py [hunk 4]
```diff
@@ -64,45 +115,33 @@ def get_baseline_system_types(rmi_b):
...
-                ]
-            )
-            if hvac_sys_count == 0:
-                raise RCTFailureException(
-                    f"Error: HVAC {hvac_b_id} does not match any baseline system type."
-                )
-            elif hvac_sys_count > 1:
-                raise RCTFailureException(
-                    f"Error: HVAC {hvac_b_id} matches to multiple baseline system types - check your RMD models"
+            if not sys_found:
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/ruleset_functions/get_baseline_system_types.py]: fixed.

**Pull request merged**